scripts/manage/deploy_application_manifest.sh

```diff
@@ -4,6 +4,12 @@ bold() {
   echo ". $(tput bold)" "$*" "$(tput sgr0)";
 }
 
+if [ ! -f "$HOME/spinnaker-for-gcp/scripts/install/properties" ]; then
```

Comment:
This happens if the user is different from the one that installed Spinnaker?

Reply:
This conditional evaluates to `true` if the file does not exist.

Reply:
This will be the case when somebody first clones the repo, and before they have pulled config from a deployment.